file5.txt

```diff
@@ -1 +1,2 @@
-helllo
+hii miko
+heyyyyy
```

Comment:
This looks not good please capitalize the first letter and avoid the repetition of "y"